packages/checkbox/src/components/FzCheckboxGroupOption.vue

```diff
@@ -32,9 +32,13 @@ import { computed } from "vue";
 import FzCheckbox from "../FzCheckbox.vue";
 import { ParentCheckbox } from "../types";
 
-const props = defineProps<ParentCheckbox & { size: "sm" | "md" }>();
+FzCheckbox.compatConfig = {
+  MODE: 3,
+};
```

Comment:
same as above

Reply:
To avoid potential problems, I have included it here as well, because I am not sure that if you set compatConifg for FzCheckbox in the production application on a page, it will address all FzCheckboxes within that page (even nested ones like this one).